src/components/Person/Person.jsx

```diff
@@ -1 +1,37 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const partnerRole = sex === 'm'
+    ? 'wife'
+    : 'husband';
```

Comment:
It would be cool to keep such data in separate constants:
```js
const MALE = 'm';
const WIFE = 'wife';
const HUSBAND = 'husband';
...
const partnerRole = sex === MALE
  ? WIFE
  : HUSABAND;
```